http/headers/Reporting-Endpoints.json

```diff
@@ -0,0 +1,51 @@
+{
+  "http": {
+    "headers": {
+      "Reporting-Endpoints": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Reporting-Endpoints",
+          "spec_url": "https://w3c.github.io/reporting/#header-field-registration",
+          "support": {
+            "chrome": {
+              "version_added": "96"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "130",
```

Comment:
Added in https://bugzilla.mozilla.org/show_bug.cgi?id=1860588

Note, isn't clear if headers are behind prefs or not (i.e. no clear signal about this as there is in IDL), but they aren't useful if the associated code gated by this isn't preference, so it makes sense to add the pref.